REQUIRE

```diff
@@ -1 +1,3 @@
 julia 0.5
+PyCall 1.12
+retriever 2.0
```

Comment:
remove retriever in this case or make it retriever 2.1

Reply:
Removed `retriever` from REQUIRE.